src/Microsoft.AspNetCore.Sockets.Client.Http/Internal/SocketClientLoggerExtensions.cs

```diff
@@ -514,7 +514,7 @@ public static void StoppingClient(this ILogger logger, string connectionId)
             }
         }
 
-        public static void ExceptionThrownFromEventHandler(this ILogger logger, string connectionId, string eventHandlerName, Exception exception)
+        public static void ExceptionThrownFromHandler(this ILogger logger, string connectionId, string eventHandlerName, Exception exception)
```

Comment:
Literally every other method name matches the field name of the delegate, except this one, so it felt right to get them in alignment. Happy to align the other way if that makes sense :)